test/e2e/performance/performance_test.go

```diff
@@ -79,11 +79,10 @@ var _ = Describe("performance test case", Serial, Label("performance"), func() {
 				}
 			}
 			// Generate Pod.IPPool annotations string and create IPv4Pool and IPV6Pool
-			ctx, cancel := context.WithTimeout(context.Background(), common.PodStartTimeout)
-			defer cancel()
+			ctx := context.TODO()
 			if frame.Info.IpV4Enabled {
-				GinkgoWriter.Printf("try to create IPv4pool: %v \n", v4PoolName)
 				v4PoolName, iPv4PoolObj = common.GenerateExampleIpv4poolObject(int(replicas))
+				GinkgoWriter.Printf("try to create IPv4pool: %v \n", v4PoolName)
```

Comment:
The previous source codes, there's no v4PoolName generated, so the line 85 can't print anything.